index.js

```diff
@@ -232,11 +232,36 @@ async function sendTransaction({to, amount, network, keyStore, password, tokenId
     }
 }
 
+
+async function getGasCost({ network, amount, tokenIdentifier, decimals }) {
+    const { payload, payment, gasLimit } = _getPayload({
+      identifier: tokenIdentifier,
+      amount,
+      decimals,
+    });
+  
+    let networkDetails = config.networks.testnet;
+    if (network === "main") networkDetails = config.networks.main;
+    const provider = new ProxyNetworkProvider(networkDetails.provider, {
+      timeout,
+    });
+    const networkConfig = await provider.getNetworkConfig();
+  
+    return {
+      gasLimit,
+      gasCostInCrypto: Number(
+        _toDecimal(gasLimit * networkConfig.MinGasPrice, decimals)
```

Comment:
@shivamangina From where are we getting this `gasLimit` here?

Reply:
We are getting from this function ``` { payload, payment, gasLimit } = _getPayload({
      identifier: tokenIdentifier,
      amount,
      decimals,
    });```
    
    this is existing function which is being used for setting gasLimit in the sendTransaction.